packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -1,15 +1,18 @@
 <template>
 	<div class="workflow-activator">
-		<el-switch
-			v-loading="loading"
-			element-loading-spinner="el-icon-loading"
-			:value="workflowActive"
-			@change="activeChanged"
-			:title="workflowActive?'Deactivate Workflow':'Activate Workflow'"
-			:disabled="disabled || loading"
-			:active-color="getActiveColor"
-			inactive-color="#8899AA">
-		</el-switch>
+		<n8n-tooltip :disabled="!disabled" placement="bottom">
+			<div slot="content">This workflow has no trigger nodes that require activation</div>
+			<el-switch
+				v-loading="loading"
+				element-loading-spinner="el-icon-loading"
+				:value="workflowActive"
+				@change="activeChanged"
+				:title="workflowActive?'Deactivate Workflow':'Activate Workflow'"
+				:disabled="disabled"
```

Comment:
just to be clear, was this an intentional change that loading state is not disabled? 
```suggestion
				:disabled="disabled | loading"
```